app/src/main/java/com/android/voyageur/ui/overview/Overview.kt

```diff
@@ -117,7 +126,53 @@ fun OverviewScreen(
       },
       modifier = Modifier.testTag("overviewScreen"),
       topBar = {
-        TopAppBar(title = { Text(text = "Your trips") }, modifier = Modifier.testTag("topBarTitle"))
+        TopAppBar(
+            title = {
+              Box(
+                  modifier = Modifier.fillMaxWidth().fillMaxHeight(),
+                  contentAlignment = Alignment.CenterStart) {
+                    if (!searchVisible) {
+                      Text(
+                          text = "Your trips",
+                          style =
+                              MaterialTheme.typography.headlineMedium.copy(
+                                  fontWeight = FontWeight.Bold),
+                          modifier = Modifier.testTag("topBarTitle"))
+                    } else {
+                      TextField(
+                          value = searchQuery,
+                          onValueChange = { searchQuery = it },
+                          placeholder = { Text("Search trips...") },
+                          modifier = Modifier.fillMaxWidth().testTag("searchField"),
+                          singleLine = true,
+                          shape = RoundedCornerShape(10.dp),
+                          keyboardOptions = KeyboardOptions(imeAction = ImeAction.Search),
+                          keyboardActions = KeyboardActions(onSearch = {}))
```

Comment:
Since our search bar filters the screen in real time, I think you could remove this line, as it seems to be doing nothing. Would there be a reason to keep it?

Reply:
Or maybe, we should think about the behaviour we want to implement.. Do we want when the user presses Enter to only see the filtered trips? I think that could be cool, wdyt?

Reply:
I just deleted it, deleting the search icon form the keyboard which now just shows a check on the Enter button. Thank you for the remark! :)